draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -64,7 +74,7 @@ This specification presents a comprehensive guide to the Nonce endpoint in OAuth
 **Nonce Endpoint**:
 :  The HTTP endpoint provided by the Nonce Issuer for the issuance of the Nonces.
 
-# Requirements
+<!-- # Requirements
```

Comment:
Why requirements is removed?
We don't have track nor agreement in the issues about this